lib/src/Uncertainty/Process/ConditionedGaussianProcess.cxx

```diff
@@ -211,7 +247,9 @@ void ConditionedGaussianProcess::save(Advocate & adv) const
 void ConditionedGaussianProcess::load(Advocate & adv)
 {
   GaussianProcess::load(adv);
+  adv.loadAttribute("inputDimension_", inputDimension_);
   adv.loadAttribute("krigingResult_", krigingResult_);
+  adv.loadAttribute("gprResult_", gprResult_);
```

Comment:
must be conditional to presence of new attributes

Reply:
indeed, thanks